design-proposals/storage-class-migration/storage-class-migration.md

```diff
@@ -0,0 +1,267 @@
+# Overview
+This design proposal extends KubeVirt APIs to enable the migration between PVCs or storage classes.
+
+## Motivation
+In some circumstances, the guest owner may want to have the option of switching the storage provider or class for a group of volumes used by an active VM. The VM must currently be turned off and recreated for this operation causing service disruptions and downtime.
+
+
+## Goals
+Design and develop an API to handle storage migrations.
+
+ ## Non  Goals
+ This feature doesn't cover the VM Live migration with RWO volumes.
```

Comment:
I can reformulate, this feature isn't for live migration with local disks. You could also use it for this, but the API is designed for migrating the storage, not the VM. The VM live migration is an implementation detail, if you want. As mentioned in the `Motivation on the approaches` section, we could also copy the storage in the destination and flip the disks when is finished. Unfortunately, this isn't possible for the reasons listed there.

Reply:
Yeah, maybe just for clarity, it would be better to formulate it as: `The feature is not directly aimed at VM Live migration with RWO volumes, but can be used to achieve the same`? .... or somehow in such way...